tests/test_endpoints.py

```diff
@@ -77,7 +76,7 @@ def test_json_error(self, mock_request):
         mock_request.return_value.status_code = 200
 
         def response_json():
-            raise JSONDecodeError('Some error message', '', 0)
+            raise ValueError
```

Comment:
I think I'd still raise the JSONDecodeError here, but leave the lines below to be expecting ValueError.

Reply:
Ah, yeah, that makes sense.